haas/client/node.py

```diff
@@ -0,0 +1,33 @@
+from haas.client.base import *
+from haas.client.client_errors import *
+
+
+class Node(ClientBase):
+    """ Consists of calls to query and manipulate node related
+    objects and relations """
+
+    def list(self, is_free):
+        """ List all nodes that HIL manages """
+        url = self.object_url('node', is_free)
+        q = requests.get(url, headers={"Authorization": "Basic %s" % self.auth})
+        if q.ok:
+            return q.json()
+        elif q.status_code == 401:
+            raise AuthenticationError("Make sure credentials match chosen authentication backend.")
```

Comment:
Could not find cases that would raise those errors. If you can help me with that, will include those.